app/views/login/login_form.html.erb

```diff
@@ -2,18 +2,22 @@
     email_prefill_value = current_users_resetting_password_email
     email_prefill_value ||= nil # because otherwise it literally shows "false" in the form's input field
 %>
+<% content_for :head do %>
+  <%= stylesheet_link_tag    "application" %>
+  <%= stylesheet_link_tag    "login_form" %>
+<% end %>
 
 <div id="login-signup-form">
     <div class="content">
-        <%= render partial: 'newflow/tab_group' , locals: { active_class: 'login--active' } %>
+        <%= render partial: 'shared/tab_group' , locals: { active_class: 'login--active' } %>
 
-        <%= newflow_login_signup_card(
+        <%= login_signup_card(
                     classes: "login-page",
                     header: (I18n.t(:"login_signup_form.login_page_header")),
                     banners: @banners,
                     show_exit_icon: true) do %>
-            <% lev_form_for :login_form, url: newflow_login_path do |f| %>
-                <% fh = NewflowFormHelper::Newflow.new(f: f, context: self, errors: @handler_result&.errors) %>
+            <% lev_form_for :login_form, url: login_path do |f| %>
+                <% fh = FormHelper::One.new(f: f, context: self, errors: @handler_result&.errors) %>
```

Comment:
The `errors` parameter here is what triggers the pink box and error message on the form.